with-better-auth/app/(auth)/sign-in.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { TextInput, Button, ScrollView, Text, StyleSheet } from "react-native";
+import { authClient } from "@/lib/auth-client";
+import { Link } from "expo-router";
+
+export default function SignIn() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleLogin = async () => {
+    await authClient.signIn.email({
+      email,
+      password,
+    });
+  };
+
+  return (
+    <ScrollView contentInsetAdjustmentBehavior="automatic">
+      <Text style={{ fontSize: 32, fontWeight: "bold", margin: 10 }}>
+        Sign In
+      </Text>
+      <TextInput
+        placeholder="Email"
+        value={email}
+        onChangeText={setEmail}
+        style={styles.input}
+      />
```

Comment:
Nit - maybe add following props; 
- inputMode
- autoCapitalize

```suggestion
<TextInput
  placeholder="Email"
  value={email}
  inputMode="email"
  autoCapitalize="none"
  onChangeText={setEmail}
  style={styles.input}
/>
```